### DISCUSSION TITLE
Dual NVidia + AMD GPU

### DISCUSSION THREAD

**Discussion by albertosottile** [LostRuins/koboldcpp#1182] [Q&A] [answered]
Currently, I have a AMD Radeon RX 5700 XT with 8 GB of VRAM. I am thinking of buying a Nvidia GeForce RTX4060 TI 16GB. 

Would it be possible to use both at the same time with koboldcpp, the Nvidia with the CUDA backend and the AMD with the Vulkan backend?

**Comment by LostRuins** [comment, marked as answer]
No,you can't mix backends. However, you can use both brands of GPU on vulkan together
